src/core/Microsoft.PowerToys.Settings.UI.Lib/HotkeySettings.cs

```diff
@@ -100,6 +102,13 @@ public override string ToString()
 
         public bool IsValid()
         {
+            // Tab and Shift+Tab are not valid settings to meet the accessibility criteria.
+            if ((!Alt && !Ctrl && !Win && Shift && Code == VKTAB)
+                || (!Alt && !Ctrl && !Win && !Shift && Code == VKTAB))
```

Comment:
The code has been modified such that I need a function which returns true only for tab and shift+tab and returns false otherwise. So these two conditions are still needed.